$NetBSD: patch-pascomp_Makefile,v 1.2 2020/09/26 11:35:14 mef Exp $

from build.log:
 ------------------------
 gcc -g -Wall -c date.c
 bison -tvd bapascal.y
 mv -f bapascal.tab.c gram.c
 mv -f bapascal.tab.h ytab.h
 chmod 444 gram.c ytab.h
 gcc -g -Wall -c gram.c
 bapascal.tab.c:217:10: fatal error: bapascal.tab.h: No such file or directory
 compilation terminated.
 ------------------------
Above is a little bit strange, change mv to cp

--- pascomp/Makefile.orig	1999-08-31 16:19:13.000000000 +0000
+++ pascomp/Makefile
@@ -23,7 +23,7 @@ date.o:
 gram.o ytab.h:  bapascal.y $(INCLS)
 	bison -tvd bapascal.y
 	mv -f bapascal.tab.c gram.c
-	mv -f bapascal.tab.h ytab.h
+	cp -f bapascal.tab.h ytab.h
 	chmod 444 gram.c ytab.h
 	$(CC) $(CFLAGS) -c gram.c
 
